docs/src/eravm/01-code-separation.md

```diff
@@ -0,0 +1,56 @@
+# Code Separation
+
+In both EVM and EraVM, contract bytecode is divided into two segments: deploy and runtime. The deploy code — also known as the constructor — runs only once when the contract is first deployed. In contrast, the runtime code executes every time the contract is called.
+
+However, on EraVM, both segments are deployed together rather than split into two separate chunks. The constructor is simply added to the contract as a standard public function, which the System Contracts invoke during deployment.
+
+Just like on the EVM, the deploy code on EraVM takes the form of a single constructor. Our compiler merges this constructor into the runtime code while generating LLVM IR, as illustrated in the minimal example below.
+
+
+
```

Comment:
Nit: unneeded empty lines

Also in other places

Reply:
It helps to visually separate logical sections.
IIRC it does not affect rendering.

Reply:
Up to you